python_dependency_verifier/python_dependency_verifier.py

```diff
@@ -118,13 +30,17 @@ def string_dependency_to_dict(string_dependency):
     and returns the following dictionary:
     [name, is_locked_to_specific_version, version_locked_to,
     latest version available from pip]"""
-    dependency_dict = re.split("<|>|=", string_dependency)
-    is_locked = False
+    dependency_list = re.split("<|>|=", string_dependency)
```

Comment:
this isn't really a "dependency list".. maybe it should be called `dependency_params`?